src/components/search-results/index.js

```diff
@@ -171,6 +154,18 @@ class SearchResults extends Component {
           } else {
             return <div/>;
           }
+        } else if (item.type === 'filter') {
+          return (
+            <VisbilitySensor key={index} onChange={(isVisible) => this.handleVisibility(isVisible, item)}>
+              <div key={index} className='gridItem'>
+                <FilterTile
+                  onYesFilter={onYesFilter}
+                  onNoFilter={() => removeTile(item.id)}
+                  description={item.filter}
+                />
+              </div>
+            </VisbilitySensor>
+          );
```

Comment:
### Query:

Each component returned in each of these conditional clauses is wrapped in the same `<VisibilitySensor/>` component. Is there a way we can DRY that out?